### PR TITLE
Ignore content-range in requests

### LINKED ISSUE #618: can Content-Range appear in a request?

### PR DESCRIPTION
**Pull request by mnot**
Fixes #618.

Note outstanding question on the issue regarding PUT.

### CODE REVIEW

File: draft-ietf-httpbis-semantics-latest.xml [hunk 1]
```diff
@@ -7777,6 +7777,11 @@ bytes=500-700,601-999
    recipient &MUST-NOT; attempt to recombine it with a stored representation.
    A proxy that receives such a message &SHOULD; forward it downstream.
 </t>
+<t>
+   A server &MUST; ignore a Content-Range header field received in a request
+   with a method for which Content-Range support is not defined. No request
+   method in this specification is defined to support Content-Range.
```
Comment (royfielding) [inline draft-ietf-httpbis-semantics-latest.xml:7783]:
```suggestion
   method in this specification is defined to support Content-Range in a request.
```

**Review by royfielding — APPROVED** [review]

**Comment by mnot** [comment]
@royfielding any thoughts about whether we should mention PUT?

**Comment by reschke** [comment]
I don't think we need to.

**Review by reschke — APPROVED** [review]

**Comment by royfielding** [comment]
I think PUT is already covered in PUT.

**Pull request merged**